deploy/kubernetes/overlays/prow-gke-release-staging-rc-1-18/kustomization.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+namespace:
+  gce-pd-csi-driver
 resources:
-- ../stable-1-18
+- ../../base
```

Comment:
stable-1-18, 1-19 removes windows specfic roles. Hence for testing staging builds, directly importing from base is easier.
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/deploy/kubernetes/overlays/stable-1-18/no_win_roles.yaml